docs/articles/documentation/test-api/selecting-page-elements/selectors/creating-selectors.md

```diff
@@ -50,10 +49,29 @@ You can initialize a selector with any of these objects.
     const element = Selector(() => {
         const storedElementId = window.localStorage.storedElementId;
 
-        return document.querySelector(storedElementId);
+        return document.getElementById(storedElementId);
     });
     ```
 
+    You can provide a function that takes arguments, and then pass serializable objects to the selector when you call it.
+
+    ```js
+    import { Selector } from 'testcafe';
+
+    const elementWithId = Selector(id => {
+        return document.getElementById(id);
+    });
+
+    fixture `My fixture`
+        .page `http://www.example.com/`;
+
+    test('My Test', async t => {
+        await t.click(elementWithId('buy'));
+    });
+    ```
+
+    If the function should always use the same argument value, you can assign it to the [options.dependencies](selector-options.md#optionsdependencies) property when the selector is created.
```

Comment:
It's true but also
1) these arguments are named
2) we can evaluate them in Selector's function in any moment